zdm/craco/loading.py

```diff
@@ -95,7 +95,7 @@ def survey_and_grid(survey_name:str='CRAFT/CRACO_1_5000',
             state_dict=None, iFRB:int=0,
                alpha_method=1, NFRB:int=100, 
                lum_func:int=2,sdir=None,nz=500,ndm=1400,
-               nbins=5):
+               nbins=5,edir=''):
     """ Load up a survey and grid for a CRACO mock dataset
 
     Args:
```

Comment:
Can you please add a description of "edir" to the "Args" directory, and also add this functionality to "real_loading.py" in the main zdm directory?